interfaces/builtin/account_control.go

```diff
@@ -39,6 +39,36 @@ const accountControlBaseDeclarationSlots = `
 `
 
 const accountControlConnectedPlugAppArmor = `
+#include <abstractions/dbus-strict>
+# Introspection of org.freedesktop.Accounts
+# do not use peer=(label=unconfined) here since this is DBus activated
```

Comment:
In general, D-Bus activation is not a reason to leave out the peer restriction from the rules. Did you encounter any problems when including that restriction in the rules?

It looks like user accounts panel in gnome-control-center is using libaccountsservice here, which in turn uses GDBusProxy objects created with `G_DBUS_PROXY_FLAGS_NONE`. So it's going to call `StartServiceByName` before trying to talk to the service, so I wouldn't expect it to cause a problem.

Further more, the `accounts-daemon.service` systemd unit uses `WantedBy=graphical.target`, so it is likely to be running anyway.

If we need to handle the activation case, it'd be useful to work out what message the settings app sends first, and add a rule to match it using `peer=(name=org.freedesktop.Accounts)`.

Reply:
I'll do some tests this evening to check if it works with `peer=(label=unconfined)`.

Reply:
I tested with `peer=(label=unconfined)` and it works the same, so I added it. Unfortunately, there was some problem in git (a rebase?) and I had to do a "--force" (sorry)